### PR TITLE
Add a hint on how to create a multi-instance from a range

### PR DESCRIPTION
**Pull request by jonathanlukas**
## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [x] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

Originates from https://camunda.slack.com/archives/C01QYD808A3/p1686567499489459

Tagging @saig0

### CODE REVIEW

File: docs/components/modeler/bpmn/multi-instance/multi-instance.md [hunk 1]
```diff
@@ -35,6 +35,8 @@ In case of a **parallel** multi-instance activity, all instances are created whe
 
 A multi-instance activity must have an `inputCollection` expression that defines the collection to iterate over (e.g. `= items`). Usually, it [accesses a variable](/components/concepts/expressions.md#access-variables) of the process instance that holds the collection. The expression is evaluated on activating the multi-instance body. It must result in an `array` of any type (e.g. `["item-1", "item-2"]`).
 
+If you need to iterate `n` times, you can use a for-loop to create an array for a range: `for i in 1..n return i`.
+
```
Comment (saig0) [inline docs/components/modeler/bpmn/multi-instance/multi-instance.md:39]: @jonathanlukas what do you think about the following suggestion?
- wrapping the tip into an Admonitions tip box
- linking to the FEEL docs

```suggestion
:::tip

If you need to iterate `n` times (like with a loop-cardinality), you can use the following expression with a [for-loop](/components/modeler/feel/language-guide/feel-control-flow.md#for-loops): `for i in 1..n return i`.

:::
```

**Review by saig0 — COMMENTED** [review]
@jonathanlukas thank you for this addition. :tada: 

Please have a look at my suggestion. If you don't like it, we can go with the original change. :+1:

**Comment by jonathanlukas** [comment]
Thank you @saig0 , this is very much appreciated. Much better than my suggestion actually.

**Review by saig0 — CHANGES_REQUESTED** [review]
@jonathanlukas please apply the suggestion also to docs for version `8.2`. This version is still not changed. :sweat_smile:

**Review by saig0 — APPROVED** [review]
@jonathanlukas LGTM. :+1:

**Pull request merged**